src/main/java/Deadline.java

```diff
@@ -0,0 +1,33 @@
+import java.text.ParseException;
+import java.util.Date;
+
+public class Deadline extends Task {
```

Comment:
Good use of the `Task` class as parent, definitely something for me to learn from.